opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -2958,7 +2958,7 @@ void CpGrid::populateLeafGridCells(Dune::cpgrid::EntityVariableBase<cpgrid::Geom
             }
             // Corner is stored in adapted_corners
             else {
-                assert(  elemLgrAndElemLgrCorner_to_adaptedCorner.count({elemLgr, preAdaptCorn}) == 1);
+                assert(  elemLgrAndElemLgrCorner_to_adaptedCorner.find({elemLgr, preAdaptCorn}) != elemLgrAndElemLgrCorner_to_adaptedCorner.end());
```

Comment:
Please store iterator returned by find and use that to get the value below. see other comment.